ihatemoney/templates/list_bills.html

```diff
@@ -98,10 +98,11 @@ <h3 class="modal-title">{{ _('Add a bill') }}</h3>
 
     {% if bills.count() > 0 %}
     <table id="bill_table" class="col table table-striped table-hover">
-        <thead><tr><th>{{ _("When?") }}</th><th>{{ _("Who paid?") }}</th><th>{{ _("For what?") }}</th><th>{{ _("For whom?") }}</th><th>{{ _("How much?") }}</th><th>{{ _("Actions") }}</th></tr></thead>
+        <thead><tr><th>{{ _("Added on") }}</th><th>{{ _("When?") }}</th><th>{{ _("Who paid?") }}</th><th>{{ _("For what?") }}</th><th>{{ _("For whom?") }}</th><th>{{ _("How much?") }}</th><th>{{ _("Actions") }}</th></tr></thead>
     <tbody>
     {% for bill in bills %}
     <tr owers="{{bill.owers|join(',','id')}}" payer="{{bill.payer.id}}">
+            <td>{{ bill.import_date }}</td>
```

Comment:
This looks good to me, except the name: should we name this `creation_date` instead?

Reply:
Probably :)

I was lacking a bit of imagination here.